docs/install-nuget-client-tools.md

```diff
@@ -55,7 +55,7 @@ Installation:
 [!INCLUDE[install-cli](includes/install-cli.md)]
 
 > [!Tip]
-> Use `nuget update -self` to update an existing nuget.exe to the latest version.
+> Use `nuget update -self` to update an existing nuget.exe to the latest version on Windows.
```

Comment:
I'd say, "On Windows, use `nuget update -self`...to someone doesn't read the whole sentence before figuring that out.